docker/build.sh

```diff
@@ -15,28 +15,29 @@
 
 IMAGES=(
   gcr.io/clusterfuzz-images/base
-  gcr.io/clusterfuzz-images/high-end
+  #gcr.io/clusterfuzz-images/high-end
```

Comment:
Are all these changes temporary for testing purposes?

Reply:
Yes, the images that do not belong in the tester workflow have been commented. Will uncomment these before merging